core/src/main/resources/hudson/model/ManageJenkinsAction/index.jelly

```diff
@@ -91,12 +91,7 @@ THE SOFTWARE.
                   <a href="${m.urlName}">
                     <div class="jenkins-section__item__icon" aria-hidden="true">
                       <l:icon src="${m.iconFileName}" class="icon" />
-                      <j:set var="badge" value="${m.badge}"/>
-                      <j:if test="${badge != null}">
-                        <div tooltip="${badge.tooltip}" class="jenkins-section__item__icon__badge alert-${badge.severity}">
-                          ${badge.text}
-                        </div>
-                      </j:if>
+                      <l:badge badge="${m.badge}"/>
```

Comment:
With my proposals this becomes
`<l:badge badge="${m.badge}" class="jenkins-section__item__icon__badge"/>`